lib/classes/ConfigSchemaHandler/index.js

```diff
@@ -170,6 +170,11 @@ class ConfigSchemaHandler {
       }
     }
 
+    // allow provider to override schema for `resources`.
+    if (options.resources) {
+      this.schema.properties.resources = options.resources;
```

Comment:
I believe we should also remove it's definition from core `configSchema.js` here:

https://github.com/serverless/serverless/blob/5f85543d8e47a9afaec6811a57e1235e82949550/lib/configSchema.js#L36-L37

Idea is that what's there is can be subject to extension and not to replacement, and here we're replacing the schema, which may lead to errors (e.g. if at some point something is added to `resources` at `configSchema.js`, it'll be ineffective.

We also should remove this line:
https://github.com/serverless/serverless/blob/5f85543d8e47a9afaec6811a57e1235e82949550/lib/classes/ConfigSchemaHandler/index.js#L35